docs/.readthedocs.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-22.04
```

Comment:
Should we go for 24.04?

Reply:
Good catch, thanks. I don't see why we shouldn't go for 24.04, I'll update the version